wbooks/src/utils/stringUtils.test.js

```diff
@@ -0,0 +1,14 @@
+import { getInitials, keepNumbers } from './stringUtils';
+
+describe('stringUtils', () => {
+  test('getInitials', () => {
+    const name = 'Santiago';
+    const res = getInitials(name);
+    expect(res).toBe(name[0]);
+  });
+  test('keepNumbers', () => {
+    const value = 24234;
+    const res = keepNumbers(value);
+    expect(typeof res).toBe('string');
```

Comment:
there's more you should test. What happens if you pass a string? If you pass a string with non-number characters?